config/routes.rb

```diff
@@ -4,6 +4,6 @@
   get 'about'       => 'pages#about'
   get 'contact'     => 'pages#contact'
   get 'volunteer'   => 'pages#volunteer'
-  get 'adopt'       => 'pages#adopt'
+  get 'adopt'       => 'rat_adoptions#index'
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.